content/overview/user-interface/top-panel.md

```diff
@@ -8,26 +8,35 @@ author: "people"
 
 The top panel is common to all darktable views and provides a number of common utility functions.
 
-![top panel](./top-panel/top-panel.png)
-
-On the right-hand-side
+# On the left-hand-side
 
 view
+![top panel - view](./top-panel/top-panel_view.png)
 : Choose which images to view in the lighttable/filmstrip, based on star rating and reject status.
 
 sort by
-: Choose the sort order of images in the lighttable/filmstrip using various image properties.
+![top panel - sort by](./top-panel/top-panel_sort-by.png)
+: Choose the property to sort by from the dropdown.
+
+sort order
+![top panel - sort order](./top-panel/top-panel_sort-order.png)
+: Switch the sort order (ascending / descending) with the arrow-toggle.
 
-On the left-hand-side (icons)
+# On the right-hand-side (icons)
 
 grouping
+![top panel - grouping](./top-panel/top-panel_grouping.png)
 : Expand or collapse grouped images
 
 thumbnail overlays
+![top panel - overlays](./top-panel/top-panel_overlays.png)
 : Define what information is displayed over thumbnails in the lighttable/filmstrip.
+: You can define different settings depending on the thumbnail size (set sizes categories in [`preferences > lighttable`](../../preferences-settings/lighttable.md))
 
 context-sensitive help
+![top panel - help](./top-panel/top-panel_help.png)
 : Click on this icon and then click on a control element to be directed to the appropriate online help page.
 
 preferences
+![top panel - preferences](./top-panel/top-panel_preferences.png)
 : Open the [preferences & settings](../../preferences-settings/_index.md) dialog
```

Comment:
I'm not convinced about this. I preferred what was there before. Perhaps if we had two screenshots - one for the left-hand side and one for the right-hand-side.

Reply:
for me it's difficult to see, which icon refers to a function:
- the star is not clear for me to define the thumbnail overlays
- I did not notice the possibility to change the sort order with the second arrow for a long time

So in my opinion it is important to split the icons (not only left and right)

Reply:
Maybe most of the problem I have with this is that all of your screenshots are different heights/widths so it looks a bit ugly. Try tidying up the screenshots so they're consistent and we'll see what it looks like.

Reply:
> I did not notice the possibility to change the sort order with the second arrow for a long time

Perhaps keep it as it was and just add this icon and a third (ascending/descending) definition (I agree it's not clear)

Reply:
I agree that the images weren't beautiful, I've now cleaned up the icons and removed the drop-downs, for me that looks fine and is informative as well

What do you think?

Reply:
maybe make the background of the icons transparent?
(and of course #inline is not necessary anymore, because I've chosen icon finally)

Reply:
no, transparent is even worse...
but it's hard to see the icons with the dark background. 
What about a lighter one, even if it is not the default-dt-style?

Reply:
Yes that's better but please can you put the previous version of the top panel image back as it had less dead space in the middle

Reply:
dead space in top panel is removed
I now put transparent icons to the right of the text, which looks very subtle and is independent of current or future themes

Reply:
hmm we were almost there as well. We should stick to icons as they're presented in the UI using the default theme for now, to be consistent with both the application and the rest of the documentation. If you want to move to a different standard please raise an issue for discussion and let's do it everywhere. Also, the current convention when referencing icons in a definition list is to put the icon first. Please see https://darktable-org.github.io/dtdocs/module-reference/processing-modules/retouch/ for an example.

Reply:
thanks for the input, I have created an issue (#145)
To follow the current convention I will change it again to the dark background for now.
If the issue is aggreed upon I will have to change all icons anyway.